style.css

```diff
@@ -137,21 +197,23 @@ button {
   border-radius: 5px;
   background-color: white;
   padding: 10px;
+  width: fit-content;
 }
 
 .project button {
   display: grid;
-  margin: 30px 160px 30px 0;
+  margin: 30px 160px 10px 0;
 }
```

Comment:
- [ ] Cool actions you did with the buttons. However, you have to follow the design according to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). You have to learn how to parse designs from the Figma. 
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/166069690-e0716ada-be1e-4c28-a51d-4d4402e4578c.png)
### Figma Template to follow
![image](https://user-images.githubusercontent.com/45460192/166069767-1568bc76-0f33-437f-878c-26d5febffcdd.png)

And you should follow these pseudo-classes below  
![image](https://user-images.githubusercontent.com/45460192/166069901-8b052d34-bf51-4ab4-b5c0-cd4929c13dc0.png)

Reply:
Thank you for pointing this out. I made it for another exercise and merged it not knowing it can cause problems. I still did not manage to fix the conflict. I don't have this piece of code locally on the current branch, 'web'. I spent hours searching and found that I need to merge this branch first and solve the problem on the 'main' branch.

***I agree with your note regarding the design. But since button animations are not a requirement for this project, I hope I can pass for now and fix it after merging this branch with the main branch.***